src/components/hand-poses.js

```diff
@@ -55,7 +75,8 @@ AFRAME.registerComponent("hand-pose", {
 AFRAME.registerComponent("hand-pose-controller", {
   multiple: true,
   schema: {
-    eventSrc: { type: "selector" }
+    eventSrc: { type: "selector" },
+    store: { type: "selector" }
```

Comment:
I'd probably call this something else, we already use the term `store` in the UI code and elsewhere for the local storage state

Reply:
👍